packages/eslint-config-4catalyzer-react/rules.js

```diff
@@ -15,6 +15,11 @@ module.exports = {
     },
     allowChildren: false,
   }],
+  // It's clearer to use required even on props with defaults to indicate
```

Comment:
disagree. if you were substituting this for flow annotations, the type would be nullable. PropTypes are for external documentation, not for how the props look inside the component

Reply:
sgrunt

Reply:
I think this makes sense, the default doesn't apply when it's null, it's unexpected that you'd get no error when the intent is to say something must be provided